ios/RNSailthruMobile.m

```diff
@@ -224,11 +224,15 @@ - (BOOL)shouldPresentInAppNotificationForMessage:(STMMessage *)message {
 }
 
 RCT_EXPORT_METHOD(presentMessageDetail:(NSDictionary *)jsDict) {
-    [self.messageStream presentMessageDetailForMessage:[RNSailthruMobile messageFromDict:jsDict]];
+    dispatch_async(dispatch_get_main_queue(), ^{
+        [self.messageStream presentMessageDetailForMessage:[RNSailthruMobile messageFromDict:jsDict]];
+    });
 }
 
 RCT_EXPORT_METHOD(dismissMessageDetail) {
-    [self.messageStream dismissMessageDetail];
+    dispatch_async(dispatch_get_main_queue(), ^{
+        [self.messageStream dismissMessageDetail];
+    });
```

Comment:
Just checking if this file replaces the need for those `Bridge` files above? I couldn't seem to find an equivalent for them (`RNCarnivalBridge.h` / `RNCarnivalBridge.m`) but maybe they're not required anymore (or I may have missed them!)?

Reply:
This file is the equivalent for RNCarnival:
RNCarnival -> RNSailthruMobile
RNCarnivalBridge -> RNSailthruMobileBridge

The RNSailthruMobileBridge files won't show up in the list of code changes because they didn't require any, I just had to remove the Carnival version.

Reply:
Ah sorry! Makes sense 🤦‍♀️